payments-core/src/main/java/com/stripe/android/IssuingCardPinService.kt

```diff
@@ -361,6 +363,7 @@ class IssuingCardPinService @VisibleForTesting internal constructor(
             stripeAccountId: String? = null,
             keyProvider: EphemeralKeyProvider
         ): IssuingCardPinService {
+            Log.w(TAG, "Please use Issuing Elements instead: https://stripe.com/docs/issuing/elements")
```

Comment:
Same here: https://stripe.com/docs/issuing/elements#issuing-elements-and-native-applications